lib/mayaUsd/ufe/Utils.h

```diff
@@ -125,6 +125,10 @@ Ufe::PathSegment dagPathToPathSegment(const MDagPath& dagPath);
 MAYAUSD_CORE_PUBLIC
 MDagPath ufeToDagPath(const Ufe::Path& ufePath);
 
+//! Verify if the UFE path is the Maya root.
```

Comment:
Hmmm, wonder about the name.  In Maya the root node is called world, and it's called that way in Maya commands and documentation as well.  Perhaps call it isMayaWorldPath()?